### ISSUE TITLE
The Dubbo registration center uses nacos, the service display is normal, and the log keeps reporting errors.

### ISSUE THREAD

**Issue by cvictory** [alibaba/nacos#1396]
服务列表里可以查询到，但是日志报错。

报错内容如下：
```
[19/06/19 04:21:36:036 CST] com.alibaba.nacos.client.naming.updater ERROR client.naming: [NA] failed to update serviceName: DEFAULT_GROUP@@configurators:org.apache.dubbo.samples.basic.api.DemoService::
java.lang.IllegalStateException: failed to req API:/nacos/v1/ns/instance/list after all servers([127.0.0.1:8848]) tried: failed to req API:http://127.0.0.1:8848/nacos/v1/ns/instance/list. code:404 msg: service not found: DEFAULT_GROUP@@configurators:org.apache.dubbo.samples.basic.api.DemoService::
	at com.alibaba.nacos.client.naming.net.NamingProxy.reqAPI(NamingProxy.java:380)
	at com.alibaba.nacos.client.naming.net.NamingProxy.reqAPI(NamingProxy.java:304)
	at com.alibaba.nacos.client.naming.net.NamingProxy.queryList(NamingProxy.java:217)
	at com.alibaba.nacos.client.naming.core.HostReactor.updateServiceNow(HostReactor.java:273)
	at com.alibaba.nacos.client.naming.core.HostReactor$UpdateTask.run(HostReactor.java:318)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:180)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
```

**Comment by paderlol** [comment]
Plz tell me
1.What's your version of Nacos and Dubbo?
2.How did you register?

**Comment by 502085563** [comment]
i have same bug ,the dubbo version is 2.7.2 and the nacos version is 1.0.0

**Comment by polarisary** [comment]
这个解决了吗？

**Comment by nkorange** [comment]
It's a bug of Dubbo 2.7.2, the Dubbo team is working on this issue. It's expected to be fixed in Dubbo 2.7.3. Refer to [dubbo#4423](https://github.com/apache/dubbo/issues/4423)
